src/output/icons.rs

```diff
@@ -544,15 +560,15 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "out"            => '\u{eb2c}',              // 
     "p12"            => Icons::KEY,              // 
     "par"            => Icons::COMPRESSED,       // 
-    "part"           => '\u{f43a}',              // 
+    "part"           => Icons::DOWNLOAD,         // 󰇚
```

Comment:
Honestly, I'm not sure I like either of these, but I'm leaning towards the latter

Reply:
This is consistent with our other partiality downloaded files